natgeo/mortar
# [SYTO-798][SYTO-799] Reskin Current Video Information on VideoPlaylist component

https://jira.natgeo.com/browse/SYTO-798
https://jira.natgeo.com/browse/SYTO-799

This PR implements the new skin in the VideoPlaylist component's left side. 

#### How to test
1. ``npm i``
2. ``gulp serve``
3. Open components section
4. Scroll down until VideoPlaylist Component
5. Check the skin is according to the design on ticket https://jira.natgeo.com/browse/SYTO-651

Note
- The kickers are not in the scope of these tickets due to they will be treated in another ticket.
- Also, in the VideoCaption component get rid of the React Truncate component due to didn't respect the boundaries of the container, sometimes exceeded the viewport width, so it's implement jquery dotdotdot, due to it's the library that used to work in this case.
- We need to wait until the colors classes are ready https://jira.natgeo.com/browse/SYTO-781, in the meantime the style will take the default element colors, but we can check the layout of the component, font, etc.

Bonus:
Add respond-to function to check if the screen respond to specific range of width.
